topics/single-cell/tutorials/scrna-case_basic-pipeline/tutorial.md

```diff
@@ -101,21 +103,38 @@ You've done all the work to make a single cell matrix, with gene counts and mito
+We've provided you with experimental data to analyse from a mouse dataset of fetal growth restriction {% cite Bacon2018 %}.
 
-You can access the data for this tutorial in multiple ways:
+> <details-title>Fun facts about the dataset</details-title>
+>
+> We are particularly keen for learners to be able to go from raw FASTQ files all the way through analysis. We aren't handing you a curated dataset that we specially modified in order for this tutorial to work. Instead, this tutorial's input dataset is the full dataset generated from the {% icon level %} [previous tutorial in this case study]({% link topics/single-cell/tutorials/scrna-case_alevin-combine-datasets/tutorial.md %}).
+> - The only difference is that in that previous tutorial, we *downsampled* the datasets in so the tutorial would run faster. Our input data, however, now uses the *full dataset*, analysed exactly the same was as in the tutorial.
+> - You can also find this dataset in the [study in Single Cell Expression Atlas](https://www.ebi.ac.uk/gxa/sc/experiments/E-MTAB-6945/results/tsne) and the [project submission](https://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-6945/).
+>
+{: .details}
 
-1. **Your own history** - If you're feeling confident that you successfully ran a workflow on all 7 samples from the previous tutorial, and that your resulting 7 AnnData objects look right (you can compare with the [answer key history](https://usegalaxy.eu/u/j.jakiela/h/all-total-samples-processed-after-alevin-into-single-object)), then you can use those! Be careful, in the previous tutorials we worked on downsampled datasets, but here we start with the object containing total data from all the samples! To avoid a million-line history, I recommend dragging the resultant datasets into a fresh history:
+{% include _includes/cyoa-choices.html option1="Import History on EU server" option2="Zenodo" option3="SCXA" default="Import-History-on-EU-server"
+       text="If you're on the EU server, (if your usegalaxy has an **EU** anywhere in the URL), then the quickest way to Get the Data for this tutorial is via importing a history. Otherwise, you can also import from Zenodo - it just might take a moment longer if you're in a live course and everyone is importing the same dataset at the same time! The SCXA is specifically for learners who are focusing on *Reusing public data*, so are not beginners." %}
 
-   {% snippet faqs/galaxy/histories_copy_dataset.md %}
+<div class="Import-History-on-EU-server" markdown="1">
 
-2. **Importing from a history** - You can import [this history](https://usegalaxy.eu/published/history?id=67ff4cea2adc574d)
+> <hands-on-title>Import History from EU server</hands-on-title>
+>
+> 1. Import the {% icon galaxy-history-input %} *Input history* by following the link below
+>
+>     {% for h in page.input_histories %}
+>       [ {{h.label}} ]( {{h.history}} )
```

Comment:
I saw you updated the label for the history in the metadata, but just so you know, you can also just add to the label here, for example:

```suggestion
>       [ {{h.label}} Input History ]( {{h.history}} )
```

will also work just fine

Reply:
Oh that is handy to know